src/jdk.javadoc/share/classes/jdk/javadoc/internal/tool/resources/javadoc.properties

```diff
@@ -277,7 +277,7 @@ main.opt.disable.line.doc.comments.desc=\
 main.Xusage.foot=\n\
 These options are non-standard and subject to change without notice.
 
-main.doclet.usage.header=\nProvided by the {0} doclet:
+main.doclet.usage.header=\nProvided by the {0} Doclet:
```

Comment:
I'm unsure about this one: `{0}` does not necessarily expand to `Standard`.

Reply:
I can change it back.
By the way I only see it used in the codebase once [here](https://github.com/openjdk/jdk/blob/20d8f58c92009a46dfb91b951e7d87b4cb8e8b41/src/jdk.javadoc/share/classes/jdk/javadoc/internal/tool/Start.java#L254) where `doclet.getName()` returns `Standard`

Reply:
> I can change it back. By the way I only see it used in the codebase once [here](https://github.com/openjdk/jdk/blob/20d8f58c92009a46dfb91b951e7d87b4cb8e8b41/src/jdk.javadoc/share/classes/jdk/javadoc/internal/tool/Start.java#L254) where `doclet.getName()` returns `Standard`

`doclet.getName()` returns a name of the doclet in use. It's not always "Standard". Check that example doclet from `jdk.javadoc/jdk.javadoc.doclet` javadoc: it returns "Example".

Reply:
I understand what you mean now. Thanks!